Fix internationalization issues on settings and prompt pages, and resolve rules file accessibility

feifei325 opened the pull request:
## Context

This PR addresses the following issues:
1. Internationalization of Menus:
    - Fixed issues where some menus on the settings and prompt pages were not correctly internationalized. All menu items are now properly translated based on the selected language.
2. Rules File Accessibility:
    - Fixed the issue where the rules file could not be clicked to open. The file is now accessible, and the functionality to view or edit the file works correctly.

## Implementation

<!--

Some description of HOW you achieved it. Perhaps give a high level description of the program flow. Did you need to refactor something? What tradeoffs did you take? Are there things in here which you’d particularly like people to pay close attention to?

-->

## Screenshots

| before | after |
| ------ | ----- |
|  <img width="421" alt="image" src="https://github.com/user-attachments/assets/d33b7fc1-8be3-43f9-8d09-b6f462d44d21" />  |   <img width="407" alt="image" src="https://github.com/user-attachments/assets/35227059-7bbb-4fb5-844d-39a78d0a6c28" />    |

## How to Test

<!--

A straightforward scenario of how to test your changes will help reviewers that are not familiar with the part of the code that you are changing but want to see it in action. This section can include a description or step-by-step instructions of how to get to the state of v2 that your change affects.

A "How To Test" section can look something like this:

- Sign in with a user with tracks
- Activate `show_awesome_cat_gifs` feature (add `?feature.show_awesome_cat_gifs=1` to your URL)
- You should see a GIF with cats dancing

-->

## Get in Touch

<!-- We'd love to have a way to chat with you about your changes if necessary. If you're in the [Roo Code Discord](https://discord.gg/roocode), please share your handle here. -->

<!-- ELLIPSIS_HIDDEN -->

----

> [!IMPORTANT]
> Fixes internationalization issues on settings and prompt pages and resolves rules file accessibility.
> 
>   - **Internationalization**:
>     - Fixed translation issues on settings and prompt pages, ensuring all menu items are translated based on the selected language.
>     - Updated translation files for multiple languages including `ko`, `pl`, `pt-BR`, `pt`, `ru`, `tr`, `zh-CN`, and `zh-TW`.
>   - **Accessibility**:
>     - Resolved issue with rules file accessibility, allowing it to be clicked and opened for viewing or editing.
> 
> <sup>This description was created by </sup>[<img alt="Ellipsis" src="https://img.shields.io/badge/Ellipsis-blue?color=175173">](https://www.ellipsis.dev?ref=RooVetGit%2FRoo-Code&utm_source=github&utm_medium=referral)<sup> for 67528845e378ed20f93b65e0d6f86fec250f7235. It will automatically update as commits are pushed.</sup>

<!-- ELLIPSIS_HIDDEN -->

feifei325 commented:
ellipsis-dev is awesome, I fixed the problem he suggested

mrubens commented:
Great! The only thing is that I removed some languages and added Vietnamese. Hopefully not too hard to resolve conflicts, or I can help in an hour or so.

feifei325 commented:
> Great! The only thing is that I removed some languages and added Vietnamese. Hopefully not too hard to resolve conflicts, or I can help in an hour or so.

OK, I'll handle the conflict resolution.
